nbconvert/filters/markdown_mistune.py

```diff
@@ -114,6 +123,15 @@ def latex_environment(self, name, text):
     def inline_math(self, text):
         return '$%s$' % text
 
-def markdown2html_mistune(source):
-    """Convert a markdown string to HTML using mistune"""
-    return MarkdownWithMath(renderer=IPythonRenderer(escape=False)).render(source)
+class Markdown2Html_Mistune(LoggingConfigurable):
+    def __init__(self, **kw):
+        super(Markdown2Html_Mistune, self).__init__(**kw)
+
+    def __call__(self, source):
+        """Convert a markdown string to HTML using mistune"""
+        renderer = IPythonRenderer(escape=False)
+        # when called appropriately by register filter, it will inherit from template's config
+        renderer.local_config = self.config
+        return MarkdownWithMath(renderer=renderer).render(source)
+
+markdown2html_mistune = Markdown2Html_Mistune
```

Comment:
A) That's API breaking-ish. as you go from `function` to `class`, so likely construct it here, (potentially wrap in a function and deprecate.